ui/src/app/components/workflows/workflow/dynamic-parts/cron-quartz-part/cron-quartz-part.component.ts

```diff
@@ -0,0 +1,213 @@
+import {AfterViewInit, Component, Input, OnInit} from '@angular/core';
```

Comment:
You said this component needs more work, so I won't review it here. Can be completed in a separate PR.

Reply:
Yes, https://github.com/AbsaOSS/hyperdrive-trigger/issues/145